### PR TITLE
Test yaml

### PR DESCRIPTION
**Pull request by dubiousOne**
Before this goes live, a config file will be need to placed in Jenkins for each job and the Opsgenie API key will need to be added as a secure credential.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ idna==2.8
 opsgenie-sdk==0.3.1
 python-dateutil==2.8.0
 pytz==2019.1
+pyyaml==5.3
 requests==2.22.0
```
Comment (stacybrock) [inline requirements.txt]: I think there's now a newer version of requests: 2.23

**Review by stacybrock — CHANGES_REQUESTED** [review]
See comment on requirements.txt re: requests library version.

**Comment by dubiousOne** [comment]
updated requirements.txt with new `requests` version

**Review by stacybrock — APPROVED** [review]

**Comment by stacybrock** [comment]
LGTM

**Pull request merged**